osrdyne/src/drivers/kubernetes.rs

```diff
@@ -208,53 +265,125 @@ impl WorkerDriver for KubernetesDriver {
+                                    MANAGED_BY_VALUE.to_owned(),
+                                );
+                                labels.insert(LABEL_WORKER_ID.to_owned(), worker_key.to_string());
+                                labels
+                            }),
+                            ..Default::default()
+                        },
+                        spec: keda::ScaledObjectSpec {
+                            scale_target_ref: keda::ScaleTargetRef {
+                                api_version: "apps/v1".to_string(),
+                                kind: "Deployment".to_string(),
+                                name: worker_deployment_name.clone(),
+                                env_source_container_name: None,
+                            },
+                            polling_interval: Some(keda.polling_interval),
+                            cooldown_period: Some(keda.cooldown_period),
+                            initial_cooldown_period: None,
+                            idle_replica_count: None,
+                            min_replica_count: Some(keda.min_replicas),
+                            max_replicas_count: Some(keda.max_replicas),
+                            triggers: vec![keda::Trigger {
+                                type_: "rabbitmq".to_string(),
+                                use_cached_metrics: keda.use_cached_metrics,
+                                metric_type: keda.metric_type.clone(),
+                                metadata: {
+                                    let mut metadata = HashMap::new();
+                                    metadata.insert("host".to_string(), keda.amqp_host.clone());
+                                    metadata.insert("protocol".to_string(), "amqp".to_string());
+                                    metadata.insert("mode".to_string(), keda.mode.clone());
+                                    metadata.insert("value".to_string(), keda.value.clone());
+                                    metadata.insert(
+                                        "activationValue".to_string(),
+                                        keda.activation_value.clone(),
+                                    );
+                                    metadata.insert("queueName".to_string(), queue_name);
+                                    metadata
+                                },
+                            }],
+                        },
+                    };
+
+                    debug!("Creating Keda ScaledObject: {:?}", scaled_object);
+
+                    kube::api::Api::namespaced(self.client.clone(), &self.options.namespace)
+                        .create(&kube::api::PostParams::default(), &scaled_object)
+                        .await
+                        .map_err(super::worker_driver::DriverError::KubernetesError)?;
+                }
+
+                // No autoscaler configured
+                _ => {}
```

Comment:
Don't you want the compiler to yell a bit if a new variant is added to `AutoscalingOptions`? I would rather be explicit on this last branch.

```suggestion
                Some(AutoscalingOptions::NoScaling) | None => {}
```